Python/Product/TestAdapter.Executor/Pytest/TestResultParser.cs

```diff
@@ -66,9 +66,13 @@ public class TestResultParser {
         /// <param name="funcname"></param>
         /// <returns></returns>
         internal static string CreatePytestId(string filename, string classname, string funcname) {
-            var classIndex = classname.LastIndexOf(".");
-            if (classIndex >= 0 && classIndex < (classname.Length - 1)) {
-                var classNameWithoutFilename = classname.Substring(classIndex + 1);
+            var classNameWithoutFilename = String.Empty;
+            var filenamePortion = filename.Replace(".py", "").Replace("\\", ".");
```

Comment:
We might need to worry about casing of .py here.
You could use Path.ChangeExtension to remove the extension
```
> using System.IO;
> var p = Path.ChangeExtension("tests\\unit\\test_statistics.py", null);
> p
"tests\\unit\\test_statistics"
> 
> var p = Path.ChangeExtension("tests\\unit\\test_statistics", null);
> p
"tests\\unit\\test_statistics"
> 
```

Reply:
good point!